src/components/pages/donation_form/Payment.vue

```diff
@@ -80,6 +81,13 @@ const paymentTypesAsOptions = computed<FormOption[]>( () => {
 		) );
 } );
 
+// An object with i18n messages for each unavailable payment type, empty strings if the payment is available
+// We only need keys for payments that show messages when disabled
+const disabledPaymentTypeMessages = computed( () => ( {
```

Comment:
If the contents are not dynamic, removing `computed` will improve performance. If you put this in a `computed` to make a type checker shut up, then just ignore this comment